exercises/practice/pig-latin/pig-latin-test.el

```diff
@@ -5,162 +5,97 @@
 ;;; Code:
 
 
-(load-file "pig-latin.el")
+;;; (load-file "pig-latin.el")
+(load-file ".meta/example.el")
```

Comment:
```suggestion
(load-file ".meta/example.el")
```

This needs to reverted back to loading the stub not the example.

Reply:
done, sorry about that.